src/Foundatio.TestHarness/Messaging/MessageBusTestBase.cs

```diff
@@ -379,6 +408,7 @@ public abstract class MessageBusTestBase : TestWithLoggingBase {
                 await CleanupMessageBusAsync(messageBus);
             }
         }
+        
```

Comment:
Some extra whitespace :)

Reply:
sorry, it's my first pull request ever ;-)

Reply:
No problem (it's not a problem, I need to update our style settings so the editor just does it for you, it's our issue not yours). Thanks again for making a pr, we appreciate it :).